pkg/registry/core/service/ipallocator/storage/storage_test.go

```diff
@@ -54,7 +54,7 @@ func newStorage(t *testing.T) (*etcd3testing.EtcdTestServer, ipallocator.Interfa
 	if err != nil {
 		t.Fatalf("unexpected error creating etcd: %v", err)
 	}
-	s, d, err := generic.NewRawStorage(etcdStorage, nil)
+	s, d, err := generic.NewRawStorage(etcdStorage.ForResource(api.Resource("serviceipallocations")), nil)
```

Comment:
nit: extract api.Resource("...") to a var and resue across this line and line 48

Reply:
Reused the ConfigForResource.